### DISCUSSION TITLE
Error in this line

### DISCUSSION THREAD

**Discussion by Ojoski** [Cyfrin/foundry-full-course-cu#599] [Q&A]
AggregatorV3Interface priceFeed = AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306);
            (,int256 price,,,) = priceFeed.latestRoundData();
This line has been returning an error. Won't let me compile. I asked "Phind" and it seemed to change the whole syntax saying this was not right (my Solidity says it too anyways) but it worked just well for out instructor. What have I missed?

**Comment by theirrationalone** [comment]
Please share the Error logs...

**Reply by Ojoski** [reply]
Sorry for the bothering you all and thank you for offering to help! I just found about a minute ago that I didn't enclose the function with the curly bracket "}" 🙈
